src/guess.js

```diff
@@ -0,0 +1,43 @@
+import compareNumbers from './number-guess.js';
+
+const generate = document.getElementById('generate');
+const guessDisplay = document.getElementById('guess-display');
+const userInput = document.getElementById('number-enter');
+const triesDisplay = document.getElementById('tries-display');
+
+let correctNumber = Math.floor(Math.random() * 20);
+let triesRemaining = 4;
+let msg = 'You have ' + triesRemaining + ' tries remaining!';
+
+triesDisplay.textContent = msg;
+
+generate.addEventListener('click', () => {
+    let userGuess = userInput.value;
+
+    triesRemaining += -1;
+
+    triesDisplay.textContent = 'You have ' + triesRemaining + ' tries remaining';
+
+    let correctResponse = 'your guess is ' + userGuess + '. You guessed the correct number!';
+    let tooLowResponse = 'your guess is ' + userGuess + '. You guessed too low, try again...'; 
+    let tooHighResponse = 'your guess is ' + userGuess + '. You guessed too high, try again...';
+    let incorrectResponse = 'Game Over...';
+    
+    if(compareNumbers(parseInt(userGuess), correctNumber) === 0){
+        guessDisplay.textContent = correctResponse;
+        generate.disabled = true;
+    } else if(compareNumbers(parseInt(userGuess), correctNumber) === -1){
+        guessDisplay.textContent = tooLowResponse;
+    } else if(compareNumbers(parseInt(userGuess), correctNumber) === 1){
+        guessDisplay.textContent = tooHighResponse;
+    }
+
+    if(triesRemaining === 0){
+        guessDisplay.textContent = incorrectResponse;
+        generate.disabled = true;
+    }
+
+  
```

Comment:
Make sure you don't have any extra spacing at the end of files like this - you shouldn't have any unnessecary white space when you submit. Since you don't have much throughout your code I'm not going to take a point off, but starting next week stuff like that will mean a point deduction!